sdv/sequential/par.py

```diff
@@ -194,6 +195,13 @@ def add_custom_constraint_class(self, class_object, class_name):
         """Error that tells the user custom constraints can't be used in the ``PARSynthesizer``."""
         raise SynthesizerInputError('The PARSynthesizer cannot accommodate custom constraints.')
 
+    def _validate_sequence_key_and_context_columns(self, sequence_key, context_columns):
+        if set(sequence_key).intersection(set(context_columns)):
```

Comment:
I think we need to use `set([sequence_key])`. If you pass a `string` to the `set()` you will get the following output:
```python
In [1]: set('abc')
Out[1]: {'a', 'b', 'c'}
```

Reply:
This function actually gets a list, the sequence_key is cast to an iterate in the `init`:
https://github.com/sdv-dev/SDV/blob/2097-context-column-cannot-be-a-sequence-key-need-better-error-message-for-this-case/sdv/sequential/par.py#L104

Reply:
I added typing to the function to make this clear:
https://github.com/sdv-dev/SDV/pull/2108/commits/ab3c87b279dc66bbd177cb6803c5881f748dd745#diff-f97bec45b00fd2731a7c766f7d860080f74d4355c407317cd4cea7ec7d38274fR199

Reply:
I added a docstring with Args to make this clear:
https://github.com/sdv-dev/SDV/pull/2108/commits/fd50434b8d1f0ff423cc9e58495c53f9ad969bf1